arrow/locales.py

```diff
@@ -121,6 +123,13 @@ class Locale:
 
     _month_name_to_ordinal: Optional[Dict[str, int]]
 
+    def __init_subclass__(cls, **kwargs: Any) -> None:
+        for locale_name in cls.names:
+            if locale_name in _locale_map:
+                raise LookupError(f"Duplicated locale name: {locale_name}")
```

Comment:
Should I handle duplicated key case like this?

Reply:
Hmm, that would be a nice way of doing it. But from my POV, shouldn't we test for duplicates as part of a merge request, so we don't end up with duplicates instead of run time?